.travis.yml

```diff
@@ -1,6 +1,9 @@
 env:
   - DOCKER_COMPOSE_VERSION=1.17.0 MOZ_HEADLESS=1
 
+stages:
+  - "Tests"
```

Comment:
If we can find a linter for the C# code, I'd like to add a pre-test stage for "Quality". But that can happen later